lib/Twig/Cache/Filesystem.php

```diff
@@ -98,6 +90,6 @@ public function write($key, $content)
      */
     public function getTimestamp($key)
     {
-        return filemtime($key);
+        return @filemtime($key);
```

Comment:
in the error case this returns false which is not allowed according to the phpdoc of `getTimestamp`